eslint.config.js

```diff
@@ -56,7 +56,12 @@ export default [
             {
               target: "**/routes/**/!(*.test).js",
               from: ["src/**/**"],
-              except: ["**/use-cases/**", "**/services/**", "**/schemas/**"],
+              except: [
+                "**/use-cases/**",
+                "**/services/**",
+                "**/schemas/**",
+                "src/common/**",
```

Comment:
The limitation was intentional, so would be good to keep it but allow what is acceptable:
```suggestion
                "src/common/logger.js",
```